QualityControl/docs/Installation.md

```diff
@@ -0,0 +1,74 @@
+# Installation
+
+Follow these steps to install and set up QCG (Web UI for O<sup>2</sup> Quality Control).
+
+## 1. Install Node.js (version 22.x or higher)
```

Comment:
I think for installation, instead of maintaining this, we should directly redirect the user to the NodeJS official download page which gives multiple options and combinations.